lib/rubygems/specification.rb

```diff
@@ -2423,6 +2423,7 @@ def test_files # :nodoc:
   # still have their default values are omitted.
 
   def to_ruby
+    require 'openssl'
```

Comment:
I think we can require it where it's needed.
https://github.com/rubygems/rubygems/blob/0dd5b8856dbecfffdb6be5b192f08804ef5ae3f4/lib/rubygems/specification.rb#L2465